bleak/backends/corebluetooth/client.py

```diff
@@ -47,10 +47,10 @@ def __init__(self, address_or_ble_device: Union[BLEDevice, str], **kwargs):
 
         if isinstance(address_or_ble_device, BLEDevice):
             self._device_info = address_or_ble_device.details
+            self._delegate = address_or_ble_device.metadata.get("delegate")
         else:
             self._device_info = None
-
-        self._device_info = None
+            self._delegate = None
```

Comment:
It could be nice to call this `self._central_manager_delegate` or just `_manager` instead to make it clear which delegate object this is since there is also a `CBPeriperalDelegate` object.

Reply:
Yes totally agree with this, updating the PR 👍